classifier_and_summarizer/summarizer/incorrect_input.py

```diff
@@ -0,0 +1,7 @@
+''' Class to define the IncorrectInputException'''
+
+
+class IncorrectInputError(Exception):
```

Comment:
A similar class exists within the TextMediaMatcher as well. Can we pull this out and make it common between the two?

Reply:
Yes! I've added a comment for it in the PR description as well.

Reply:
Can we make those changes here as part of this PR itself? Or alternatively, add a new Issue or TODO for the same.

Reply:
Updated in this PR!